packages/swapper/src/swappers/LifiSwapper/getTradeQuote/getTradeQuote.ts

```diff
@@ -243,10 +247,11 @@ export async function getTrade(
 
       return {
         id: selectedLifiRoute.id,
-        // This isn't a mistake - with Li.Fi, we can never go with our full-on intent of rate vs. quotes. As soon as a wallet is connected, we get a *quote*
-        // even though we're lying and saying this is a rate. With the "rate" containing a receiveAddress, a quote will *not* be fired at pre-sign time, which
-        // ensures users aren't rugged with routes that aren't available anymore when going from input to confirm
-        receiveAddress,
+        // TODO(gomes): when https://github.com/shapeshift/web/pull/8309 goes in, this goes out
+        // We do need receiveAddress in *input* to send it as fromAddress for routes req for more reliable rates, but with receiveAddress currently being the quotes/rates discriminator,
+        // we need to exclude it from method in *output*
```

Comment:
i.e whichever goes in first will have to be rebase and mergefix to accomodate for the new `quoteOrRate` discriminator, while keeping the `lifiTools` diff